src/main/docker/artemis/artemis.yml

```diff
@@ -0,0 +1,45 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Artemis base service
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    artemis-app:
+        container_name: artemis-app
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
+        # TODO: add support for armv8
+        platform: amd64
+        # TODO: add hint in documentation or here: build vs pre-built image from registry
+        # TODO: change to develop again before merging
+        image: ghcr.io/ls1intum/artemis:pr-5915
+        build:
+            context: ../../../..
+            dockerfile: src/main/docker/artemis/Dockerfile
+            pull: true
+        volumes:
+            - artemis-data:/opt/artemis/data
+        env_file:
+            - ../../resources/config/docker-application-local.env
+        # environments can also be used for custom overrides (alternative to application-local.yml)
+        # the following environments are necessary for docker images orchestrated by docker compose
+        environment:
+            _JAVA_OPTIONS: -Xmx5120m -Xms2560m
+            SPRING_PROFILES_ACTIVE: artemis,scheduling,athene,prod,docker
+        # TODO: add a restart at a certain stage for prod systems for sure, not sure about dev systems?, discuss
+        # restart: unless-stopped
```

Comment:
@b-fein and @dfuchss: What are your opinions to have the `restart` key with `unless-stopped` as a default also for dev systems? Or should this just be included in prod systems? I'm not sure if it might bug developers, for example when it restarts again and again because of an error.

Reply:
I’d prefer a manual restart. That should be easy with arrow up in the console to run `docker compose up -d` again.

If you want to add a restart policy, then I’d not use `unless-stopped` but `restart: on-failure:3` to limit the maximum number of restart attempts while still allowing an initial crash, e.g., if the database was not ready yet. If the error persists for more than a few tries the issue is probably bigger. Fixing it then is easier without a container restarting again and again taking up resources.